cmd/manager/main.go

```diff
@@ -36,7 +44,13 @@ var (
 	metricsPort         int32 = 8383
 	operatorMetricsPort int32 = 8686
 )
-var log = logf.Log.WithName("cmd")
+
+var log = logf.Log.WithName(version.OperatorName)
+
+const ManagedVeleroOperatorNamespace = "openshift-velero"
+
+// supportedPlatforms is the list of platform supported by the operator
```

Comment:
Does this mean "Where the operator can run" or "Where the operator will provision Velero" ?

Reply:
Where the operator will run. If it's not in the list, then on L169 it checks and will exit 1, as it's a fatal error.

Reply:
Basically, this won't work on CRC, Bare Metal, or any other cloud provider right now. However, it could be expanded to work on all three major providers fairly easily.

Reply:
Not that we need to worry about that for OSD, but why wouldn't it work on bare metal?

Reply:
Right now it relies on the OpenShift infrastructure config to determine which AWS region to create the bucket in, and it will use EC2 snapshotting for PV backups.

Velero itself could work on bare metal, it’s just this operator that’s opinionated for OSD.

Reply:
That makes sense! Cheers.